README.md

```diff
@@ -203,8 +187,6 @@ The following settings are stored in `template_config.yml`.
   pre_job_template      holds name and a path for a template to be included to run before jobs.
   post_job_template     holds name and a path for a template to be included to run after jobs.
   lint_requirements     Boolean (defaults True) to enable upper bound check on requirements.txt
-  use_unified_docs      Boolean (defaults False) to enable the unified docs CI (see [pulp-docs](https://github.com/pulp/pulp-docs).
```

Comment:
This may actually still be helpful.

Reply:
Can you elaborate? The only purpose of this was:
```
{%- if use_unified_docs %}
  # Unified Docs Build
  pulp-docs build
{%- endif %}
```
But, we did decide to use only new docs, did not we?

Reply:
But not everyone may have them.
I think it's still new docs or no docs.

Reply:
Do you think that running `pulp-docs build` will have any negative impact on plugins that have no docs, @pedro-psb?

Reply:
It is still able to build if a plugin does not contain any docs dir all.